### DISCUSSION TITLE
job reference documentation needs update (fscrawler 2.8)

### DISCUSSION THREAD

**Discussion by cbb-colab** [dadoonet/fscrawler#1342] [Q&A]
This is just to improve the documentation:

using fscrawler 2.8 (from docker image: dadoonet/fscrawler:noocr) I created my first job

I took the job description from

https://fscrawler.readthedocs.io/en/latest/admin/fs/index.html

An created my job based on this example.

a)
On running it gave a warning on one setting to be a deprecated setting,.

The offending item for me was
pdf_ocr: false

which I changed then to
ocr.pdf_strategy: no_ocr

as proposed

b)
Also in the config entry
follow_symlink: false

option was missing, which I needed to change and needed to look up the option somewhere else

**Comment by dadoonet** [comment]
That's great! I opened #1344 as a documentation issue. Would you like to contribute a PR to fix the doc?
